src/ModeCouplingTheory.jl

```diff
@@ -7,14 +7,14 @@ module ModeCouplingTheory
     import LinearSolve
     
     export solve, TimeDoublingSolver, EulerSolver
-    export ModeCouplingKernel, MultiComponentModeCouplingKernel, ExponentiallyDecayingKernel, SchematicDiagonalKernel, SchematicF123Kernel, SchematicF1Kernel, SchematicF2Kernel, SchematicMatrixKernel
-    export SjogrenKernel, TaggedSchematicF2Kernel, TaggedModeCouplingKernel, TaggedMultiComponentModeCouplingKernel
+    export ModeCouplingKernel, MultiComponentModeCouplingKernel, dDimModeCouplingKernel,  ExponentiallyDecayingKernel, SchematicDiagonalKernel, SchematicF123Kernel, SchematicF1Kernel, SchematicF2Kernel, SchematicMatrixKernel
```

Comment:
No need to export these
```suggestion
    export ModeCouplingKernel, MultiComponentModeCouplingKernel,  ExponentiallyDecayingKernel, SchematicDiagonalKernel, SchematicF123Kernel, SchematicF1Kernel, SchematicF2Kernel, SchematicMatrixKernel
```